src/BasePlatform.ts

```diff
@@ -457,4 +457,13 @@ export default abstract class BasePlatform {
     public getOidcClientState(): string {
         return "";
     }
+
+    /**
+     * The URL to return to after a successful OIDC authentication
+     */
+    public getOidcCallbackUrl(): URL {
+        const url = new URL(window.location.href);
+        url.hash = "";
```

Comment:
can you explain why we don't need `fragmentAfterLogin` here? Isn't it used to make sure that links to specific rooms/etc are preserved through the login flow?

Reply:
Because we can't do that with OIDC anyway, the redirectUrl check is exact, you'd have to specify all possible fragments in the valid redirectUrls are client registration time

Reply:
huh, fair enough!

Reply:
```suggestion
        // The redirect URL has to exactly match that registered at the OIDC server, so
        // ensure that the fragment part of the URL is empty.
        url.hash = "";
```